src/bref-local

```diff
@@ -18,6 +18,16 @@ if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
 
 $handler = $argv[1] ?? '';
 $data = $argv[2] ?? null;
+$opts = getopt('', ['path:']);
+
+if (isset($opts['path'])) {
+    if (! file_exists($opts['path'])) {
```

Comment:
This should be changed to `! is_file(...)` because `file_exists` also returns true for directories.